bubblewrap.c

```diff
@@ -1327,6 +1333,10 @@ main (int    argc,
   if (!is_privileged)
     opt_unshare_user = TRUE;
 
+  if (opt_unshare_user_try &&
+      stat ("/proc/self/ns/user", &sbuf) == 0)
```

Comment:
EINVAL can happen for a million reasons though, its not very robust to assume a particular reason.

The patch above fixes Arch, and debian works by virtue of it being setuid there.